eskip/string.go

```diff
@@ -80,7 +81,7 @@ func argsString(args []interface{}) string {
 	return strings.Join(sargs, ", ")
 }
 
-func (r *Route) predicateString(sortPredicates bool) string {
+func (r *Route) predicateString(sortPredicates, sortHeaderPredicateValues bool) string {
```

Comment:
why do we need 2 args or do we need to call this sometimes with `true, false` ?

Reply:
we can pass a pointer to PrettyPrintInfo